### PR TITLE
Milestone 2: Create middle section

### PR DESCRIPTION
**Pull request by GutemaG**
# Create a middle section
- Created works-section
- Created about-section
-  Add Poppins google fonts.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,5 +53,234 @@ <h2>LET’S CONNECT</h2>
...
+        <div class="card">
+          <div class="portfolio-snapshoot">
+            <img
+              src="assets/images/snapshoot-portfolio-1.png"
+              alt="portfolio-1"
+            />
+          </div>
+          <div class="left-block">
+            <div class="portfolio-info">
+              <p class="portfolio-title">TONIC</p>
```
Comment (ichala) [inline index.html]: - [x] for best SEO performance, the title of the cards (e.g., "Tonic") should use a header tag `hx` (h1..h6) so kindly fix all of cards ,  you can read more about it [here](https://www.reliablesoft.net/h1-tag/)

File: index.html [hunk 2]
```diff
@@ -47,5 +53,234 @@ <h2>LET’S CONNECT</h2>
...
+            </div>
+            <div class="actions">
+              <button class="action action-text btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="top">
+          <p class="about-my-self-title">About Myself</p>
```
Comment (ichala) [inline index.html]: - [x] for best SEO performance, the title of the sections (e.g., "About Myself") should use a header tag `hx` (h1..h6) so kindly fix it ,  you can read more about it [here](https://www.reliablesoft.net/h1-tag/)

File: index.html [hunk 3]
```diff
@@ -47,5 +53,234 @@ <h2>LET’S CONNECT</h2>
...
+            <hr class="separator" />
+            <div class="skill-wrapper">
+              <ul class="skill-list">
+                <li class="skill">
+                  <span class="skill-logo"></span>
+                  <span class="skill-name"></span>
+                </li>
+              </ul>
+            </div>
+          </div>
```
Comment (ichala) [inline index.html]: - [ ] to be semantically correct, the skills list (eg : Languages, Framework, etc ..) should be grouped by `<ul>` list and `<li>` per each skill , kindly fix it 🙏

File: index.html [hunk 4]
```diff
@@ -47,5 +53,234 @@ <h2>LET’S CONNECT</h2>
...
+                <li class="language">
+                  <span class="langauge-logo">
+                    <img src="assets/images/js.png" alt="js" />
+                  </span>
+                  <span class="language-name"> Javascript </span>
+                </li>
+                <li class="language">
+                  <span>
+                    <img src="assets/images/html elipse.png" alt="html" />
+                  </span>
```
Comment (ichala) [inline index.html]: - [ ] to match Figma design kindly add some paddings/margins between the skill icon and the container of the skill , and kindly don't forget to add the bottom border line below each one of them

**_figma design :_**
image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @GutemaG  
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Great Design✔️ 
- Good Use Of flex-box & Grid ✔️
- Good PR title & Description ✔️



 ## Required Changes ♻️

- [ ] 🛑 **_Check The Comment Below_**

## Optional suggestions

-  N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (GutemaG) [reply index.html]: Hi @ichala, I tried to correct it can you check it, please?  
- If you have material on semantics please share it with me.

**Comment by GutemaG** [comment]
@ichala, Please Can you check now, I tried to correct all comments you suggested.

**Comment by ichala** [comment]
> @ichala, Please Can you check now, I tried to correct all comments you suggested.
> 

@GutemaG as soon as you ready you can request another review from your dashboard 🙏

**Comment by GutemaG** [comment]
@ichala, Ok thank you! do you think it is good? or any suggestions.

File: index.html [hunk 5]
```diff
@@ -47,5 +53,235 @@ <h2>LET’S CONNECT</h2>
...
+        <div class="card">
+          <div class="portfolio-snapshoot">
+            <img
+              src="assets/images/snapshoot-portfolio-1.png"
+              alt="portfolio-1"
+            />
+          </div>
+          <div class="left-block">
+            <div class="portfolio-info">
+              <p class="portfolio-title">TONIC</p>
```
Comment (christianbiring1) [inline index.html]: - [ ] As said by the previous reviewer, Kindly make sure your project title uses the header tag `hx`

File: index.html [hunk 6]
```diff
@@ -47,5 +53,235 @@ <h2>LET’S CONNECT</h2>
...
+        <div class="card">
+          <div class="portfolio-snapshoot">
+            <img
+              src="assets/images/snapshoot-portfolio-2.png"
+              alt="portfolio-1"
+            />
+          </div>
+          <div class="left-block">
+            <div class="portfolio-info">
+              <p class="portfolio-title">Multi-Post Stories</p>
```
Comment (christianbiring1) [inline index.html]: - [ ] As said by the previous reviewer, Kindly make sure your project title uses the header tag `hx`

File: index.html [hunk 7]
```diff
@@ -47,5 +53,235 @@ <h2>LET’S CONNECT</h2>
...
+        <div class="card">
+          <div class="portfolio-snapshoot">
+            <img
+              src="assets/images/snapshoot-portfolio-1.png"
+              alt="portfolio-1"
+            />
+          </div>
+          <div class="left-block">
+            <div class="portfolio-info">
+              <p class="portfolio-title">Multi-Post Stories</p>
```
Comment (christianbiring1) [inline index.html]: - [ ] As said by the previous reviewer, Kindly make sure your project title uses the header tag `hx`

File: style.css [hunk 8]
```diff
@@ -59,3 +64,327 @@
   margin: 0;
   padding: 0;
 }
+
+.main {
+  background: #f7f7f9;
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: center;
+}
```
Comment (christianbiring1) [inline style.css]: [OPTIONAL]
- [ ] Also you may consider adding some spacing (margin, padding) to your page, especially on the left side.
As you cam see how your design looks like :point_down: 
image

Figma design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @GutemaG , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

# REQUEST CHANGES :recycle: 
soclose


 ## Highlights
- [ ] All linters  checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GutemaG** [comment]
@christianbiring1 can you check it again, please?

**Comment by GutemaG** [comment]
Hi @christianbiring1, I need to be sure for next submission because the next request for code re review will be my last and I don't want to repeat the week. Please, can you check it for me.

**Review by Debas-31 — APPROVED** [review]
> ## Hi @GutemaG :wave: 

# Status: Approved :green_circle: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- GitHub flow :heavy_check_mark: 
- Project working :heavy_check_mark: 
- Commit message :heavy_check_mark: 
- PR descriptive and summary :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**